app/i18n/it/conf.php

```diff
@@ -149,6 +149,10 @@
 		'image_url' => 'Image URL',	// TODO
 		'name' => 'Nome',
 		'no_filter' => 'Nessun filtro',
+		'no_queries' => array(
+			'_' => 'No user queries are saved yet.',	// TODO
```

Comment:
```suggestion
			'_' => 'Nessuna ricerca personale è stata ancora inserita',
```
@math-GH I saw that the Italian file was not edited.
I guess there was a problem and I postpone the translation